chirpr.py

```diff
@@ -20,13 +20,42 @@ def users():
 def delete_user(user_id):
     db_access.delete_user(user_id)
     return redirect(url_for('users'))
+ 
 
-
-@app.route('/admin/user/add', methods=['POST'])
+@app.route('/user/add', methods=['POST'])
 def add_user():
     handle = request.form.get('handle')
-    db_access.add_user(handle)
+    password = request.form.get('password')
+    data = db_access.add_user(handle, password)
+    if data != None:
+        return redirect(url_for('account', success='0'))
+    tup = db_access.sign_in(handle, password)
+    if tup[0] == True:
+        session['user'] = tup[1]
```

Comment:
The card we are working on is signing up. The sign-in card is next, so should not be in this pull request. Please remove sign-in functionality.